docs/guides/deployment/kubernetes.md

```diff
@@ -94,8 +94,36 @@ Let's start by creating a new cluster. If you already have one, skip ahead to th
       creation failed: Constraint constraints/compute.vmExternalIpAccess violated for project 000000000000. Add instance projects/<GCP-PROJECT-NAME>/zones/us-east1-b/instances/gke-gke-guide-1-default-pool-c369c84d-wcfl to the constraint to use external IP with it."
       ```
 
-<!-- === "Azure"
-    TODO -->
+=== "Azure"
+
+    You can quickly create an AKS cluster using the [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/get-started-with-azure-cli), or use the Cloud Shell directly from the Azure portal [shell.azure.com](https://shell.azure.com).
+
+    First, authenticate to Azure if not already done.
+
+    ```bash
+      az login
+    ```
+
+    Next, deploy the cluster - this command will take ~4 minutes to complete. Once the cluster has been created, authenticate to the cluster.
+
+    ```bash
+
+      # Create a Resource Group at the desired location, e.g. westus
+      az group create --name <RESOURCE-GROUP-NAME> --location <LOCATION>
+
+      # Create a kubernetes cluster with default kubernetes version, default SKU load balancer (Standard) and default vm set type (VirtualMachineScaleSets)
+      az aks create --resource-group <RESOURCE-GROUP-NAME> --name <CLUSTER-NAME>
+
+      # Configure kubectl to connect to your Kubernetes cluster
+      az aks get-credentials --resource-group <RESOURCE-GROUP-NAME> --name <CLUSTER-NAME>
+
+      # Verify the connection by listing the cluster nodes
+      kubectl get nodes
+    ```
+
+    !!! Note "Note"
```

Comment:
This feels more like a warning to me.

Reply:
Sure, worth noting I've seen Azure guides throw it in regardless, it won't do anything if you have an SSH key, but I debating with myself if it's worth just adding it, or add and justifying, or bring it up here.

Reply:
PS: The reason I had it as a Note is AKS will suggest it anyway, e.g.:

```
islam@islams-MacBook-Pro ~ % az aks create -g islam-docs-aks -n islam-prefect-cluster       
An RSA key file or key value must be supplied to SSH Key Value. You can use --generate-ssh-keys to let CLI generate one for you
```

Reply:
I think its worth bringing up in a callout as you've done here. If you make it a warning instead of a note, it will be orange and include a caution icon, like this: 

![Screenshot 2023-08-31 at 4 45 03 PM](https://github.com/PrefectHQ/prefect/assets/3407835/e943c692-ef90-48a3-82c7-f598d80fc9a2)

Reply:
Ah I see! In that case, you may not need it at all.

Reply:
Ok cool, I'll remove the 'note' altogether then.

Here's the Azure AKS quick-start where they don't even bring it up but just use it.
https://learn.microsoft.com/en-us/azure/aks/learn/quick-kubernetes-deploy-cli#connect-to-the-cluster